doc/config-options.rst

```diff
@@ -198,10 +198,10 @@ a block when :ref:`friendly_fire_on_grief` is enabled.
 teamswitch_interval
 +++++++++++++++++++
 
-Forces players to wait a number of minutes before being able to switch back
+Forces players to wait a set duration before being able to switch back
 again after they switched teams.
 
-0 disables the cooldown.
+"0sec" disables the cooldown.
```

Comment:
hm, I'd love if just a pain "0" still worked. After all, it doesn't matter if it's years or months if it's zero.

Reply:
That'd work actually. If you pass it digits it'll take it as seconds. https://github.com/piqueserver/piqueserver/blob/6c7eadcb86ebe4a0ebf05d1427fc5c3ef9a1e6d4/piqueserver/config.py#L294-L300

Reply:
For the sake of consistence made everything use duration.